src/managed/CommonManaged.props

```diff
@@ -26,4 +32,5 @@
     <Reference Include="System.Core" />
   </ItemGroup>
 
+  <Import Condition="Exists('$(RepoRoot)Tools/versioning.targets')" Project="$(RepoRoot)Tools/versioning.targets" />
```

Comment:
The intent is to have similar property values defined between this, and other repo's.  ie, copyright info specified for a CoreFx binary would be the same as copyright info for a core-setup binary.  BuildTools is the common infrastructure used to define this.  However, your point is valid, all we really need to import is versioning.props, not all of versioning.targets.  Versioning.props will be imported by versioning.targets (which is used by other repos) and, also imported for the `dotnet build` here.